website/src/data/users.tsx

```diff
@@ -42,9 +42,9 @@ const Users: User[] = [
     tags: ['featured','dotnetCsharp','cosmosdb', 'aca', 'bicep'],
   },
   {
-    title: 'PyCon Workshop - Django On Azure',
```

Comment:
This is not the one used for this year's PyCon, so the title may be confusing. The distinction of this repo is that Anthony configures it a bit more for scalability than the other ones (using Azure Storage for static assets, eg).